harrison/util/test_inspectlib.py

```diff
@@ -28,3 +29,8 @@ def test_stack_frame_info(self):
 
         with self.assertRaises(ValueError):
             stack_frame_test_func(0)
+
+    @mock.patch('inspect.getmodule')
+    def test_getmodule_return_None_works(self, mock_getmodule):
```

Comment:
More like `test_stack_frame_info_works_when_module_can_not_be_identified`. The name here suggests that we're testing that `getmodule` successfully returns `None`; a year from now when someone is trying to figure out why there's this new test failure, this name would only confuse them.